docs/vereisten/verantwoordingsplicht_rechtmatigheid.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Verantwoordingsplicht voor de rechtmatigheid van de verwerking
+toelichting: Bij het verwerken van persoonsgegevens voor algoritmes en AI-systemen moeten de verantwoordelijkheden kunnen aantonen dat de verwerking rechtmatig is.
+status_vereiste: 
+ - Geldend
+levenscyclus: 
+- ontwerp
+- dataverkenning-en-datapreparatie
+- ontwikkelen
+- monitoren
+- archiveren
+bouwblok: 
+- governance
+- privacy-en-gegevensbescherming
+rekenregels: 
+- niet-impactvol: ja
+- impactvol: Ja
+machine-learning: 
+- niet-impactvol: ja
+- impactvol: Ja
+- hoog-risico: Ja
+generatieve-ai: 
+- niet-impactvol: ja
+- impactvol: Ja
+- hoog-risico: Ja
+---
+
+<!-- tags -->
+## Vereiste
+
+De verantwoordelijkheden moeten bij de verwerking van persoonsgegevens door algoritmes en AI-systemen kunnen aantonen dat de verwerkingen rechtmatig plaatsvindinden
+- Dit betekent concreet dat de volgende punten aangetoond kunnen worden:
+ - Rechtmatigheid, behoorlijkheid en transparantie.
+ - Doelbinding.
+ - Minimale gegevensverwerking.
+ - Juistheid.
+ - Opslagbeperking.
+ - Integriteit en vertrouwelijkheid
+Een aandachtpunt daarbij is dat de rechtmatigheid van de verwerking ten opzicht van andere gerelateerde wetgeving zoals de AI Act en de Algemene wet gelijke behandeling ook moeten kunnen worden aangetoond voor zover de rechtmatigheid van de verwerking onder de AVG daarvan afhankelijk is.
+
+## Toelichting 
+
+Bij het verwerken van persoonsgegevens voor algoritmes en AI-systemen moeten de verantwoordelijkheden duidelijk beschreven en toegewezen zijn.
+De verwerkingsverantwoordelijke is degene die ervoor zorgt dat deze verantwoordelijkheden worden nageleefd en kan dit aantonen, wat bekend staat als de verantwoordingsplicht.
+Deze maatregelen zijn essentieel om de naleving van regelgeving met betrekking tot gegevensbescherming te waarborgen en het vertrouwen van gebruikers in de verwerking van hun gegevens te vergroten.
+
+## Bronnen 
+
+| Bron                        |
+|-------------|
+|Artikel 5 AVG|
+
+## Wanneer van toepassing? 
+Deze vereiste is van toepassing op alle algoritmen die persoonsgegevens verwerken.
+
+
+## Risico 
+
+- De verwerking van persoonsgegevens moet niet alleen rechtmatig zijn, maar moet aantoonbaar rechtmatig plaatsvindinden. Dit vraagt een niveau van documentatie en onderbouwing waar veel projecten niet op voorbereid zijn.
+- Aantoonbare compliance kan lastig gemaakt worden door gebrekkige transparantie door leveranciers.
```

Comment:
Voorstel:
Als persoonsgegevens niet correct worden verwerkt of beveiligd vanwege het niet naleven van verantwoordelijkheden, kunnen individuen worden blootgesteld aan privacyrisico's. 
Als een organisatie niet kan aantonen dat zij de verantwoordelijkheden correct heeft toegewezen, kan dit leiden tot boetes, juridische problemen en verlies van vertrouwen bij burgers. 
Aantoonbare compliance kan lastig gemaakt worden door gebrekkige transparantie door leveranciers.

Reply:
Waar komt vandaan dat artikel 5 AVG gaat over het toekennen van verantwoordelijkheid? Artikel 5 gaat echt verder en het is juist die niet op de wet gebaseerde focus voor een heel klein onderdeel van artikel 5 die ik wil rechtzetten met mijn voorstel.

Reply:
Nu ik mijn reactie teruglees komt het misschien wat botter over dan ik 'm bedoeld heb. Ik merk een niet uitgesproken en niet onderbouwde behoefte om specifiek iets te zeggen over hoofdstuk 4 van de AVG, of specifieker artikel 24 t/m 29 en 32 denk ik. Omdat ik geen onderbouwing krijg snap ik dat niet. Is er een behoefte om de impact van artikel 5 wat te temperen omdat het anders lezers afschrikt? Bedoel je dat het toekennen van verantwoordelijkheid de belangrijkste kern is van artikel 5?

De risico's die bij deze verplichting horen zijn vrij droog. Het is een verplichting die de bewijslast omdraait van toezichthouder die overtredingen moet aantonen naar verwerkingsverantwoordelijken die compliance moet kunnen aantonen. De risico's zijn daardoor vrij saai/administratief. Ik zal een voorstel doen om 'm te vertalen naar organisatie-/projectrisico's. Maar ik denk dat het voor de discussie handig is om te weten wat de wens is om toch meer focus te leggen op de verdeling van verantwoordelijkheid. ZOlang ik die wens niet snap kan ik 'm ook niet meenemen in een nieuw voorstel.

Reply:
Ik heb geprobeerd om een opmerking toe tevoegen over het verdelen van verantwoordelijkheid, maar echt lekker past het niet bij de risico's. Komt dit wel tegemoed aan wat je verwacht over de verdeling van verantwoordelijkheid?

Reply:
Ik kan de opmerking niet terugvinden? Anyways, ik heb het gevoel dat we een beetje langs elkaar heen praten / elkaar niet helemaal begrijpen. Wellicht is het handiger als ik zelf even een voorstel doe voor de risico sectie, want daar hangt het pull request nu op:

> Als een organisatie niet kan aantonen dat de verwerking van persoonsgegevens rechtmatig is, kan dit leiden tot boetes, juridische problemen en verlies van vertrouwen bij burgers

Wat vind je van deze tekst? Dan zouden we daar het pull request mee kunnen finalizen.

Reply:
Mijn voorstel staat in [831ef65](https://github.com/MinBZK/Algoritmekader/pull/71/commits/831ef65186b8f34298be4dafc4d0ab5409a4e409).

Ik kan me niet vinden in je laatste voorstel. Het gevolg van het niet kunnen aantonen dat de verwerking rechtmatig is is dat dat onrechtmatig is en de verwerking niet mag plaatsvinden. Ik denk niet dat het verstandig is om vanuit het algoritmekader te adviseren dat je het doorzetten van een onrechtmatige verwerking een risico is dat je kan nemen. Maar ook dat is een stijlkeuze waar anderen een andere mening over kunnen hebben.